app/controllers/pages_controller.rb

```diff
@@ -62,7 +62,7 @@ def get_job(pbsid, cluster)
       Jobstatusdata.new({name: name, attribs: attribs}, cluster.id, true)
 
     rescue
-      "[{\"name\":\"#{pbsid}\",\"error\":\"Job data expired or invalid.\"}]"
+      { name: pbsid, error: "INVALID" }
```

Comment:
Is there any reason why error says "INVALID" instead of "Job data expired or invalid." or even changing this to:

```ruby
rescue => e
  { name: pbsid, error: e.message }
end
```